src/rbd.cc

```diff
@@ -100,9 +100,9 @@ void usage()
 "  create [--order <bits>] [--image-features <features>] [--image-shared]\n"
 "         --size <M/G/T> <image-name>          create an empty image\n"
 "  clone [--order <bits>] [--image-features <features>] [--image-shared]\n"
-"        <parentsnap> <clonename>              clone a snapshot into a COW\n"
-"                                              child image\n"
-"  children <snap-name>                        display children of snapshot\n"
+"        --snap <parentsnap> <image-name> <clonename>\n"
+"                                              clone a snapshot into a COW child image\n"
+"  children --snap <snap-name> <image-name>    display children of snapshot\n"
```

Comment:
Since ```snap-name``` is documented as ```[pool/]name[@snap]``` in the help, perhaps we should use just ```snap``` when the intent is only a snapshot name within an image.